packages/os/host-ctr-registry-config

```diff
@@ -0,0 +1,6 @@
+{{~#if settings.container-registry.mirrors}}
+{{~#each settings.container-registry.mirrors}}
+[mirrors."{{@key}}"]
+endpoints = [{{join_array ", " this }}]
+{{~/each}}
+{{~/if}}
```

Comment:
Is TOML what we're preferring here since it matches containerd's config file format?  Since this isn't exposed we could do something different, and Go has built-in JSON support rather than requiring an external library.

Reply:
I tried rendering it as JSON but didn't work out. See https://github.com/bottlerocket-os/bottlerocket/pull/1629#issuecomment-887994906

Reply:
Thanks for testing that; TOML seems like a reasonable choice for now.